pkg/apiserver/rest/dualwriter.go

```diff
@@ -174,17 +174,16 @@ func SetDualWritingMode(
 			return nil, errDualWriterSetCurrentMode
 		}
 	}
-	// #TODO enable this check when we have a flag/config for setting mode 1 as the desired mode
-	// if features.IsEnabledGlobally(featuremgmt.FlagDualWritePlaylistsMode1) && (currentMode == Mode2) {
-	// 	// This is where we go through the different gates to allow the instance to migrate from mode 2 to mode 1.
-	// 	// There are none between mode 1 and mode 2
-	// 	currentMode = Mode1
-
-	// 	err := kvs.Set(ctx, entity, fmt.Sprint(currentMode))
-	// 	if err != nil {
-	// 		return nil, errDualWriterSetCurrentMode
-	// 	}
-	// }
+	if (desiredMode == Mode1) && (currentMode == Mode2) {
+		// This is where we go through the different gates to allow the instance to migrate from mode 2 to mode 1.
+		// There are none between mode 1 and mode 2
+		currentMode = Mode1
```

Comment:
It's not always going to be safe to move from a higher to a lower mode without losing data, specifically if we wanted to move from mode 4 to mode 2 or lower we'd need to sync data from k8s back to the legacy db to avoid losing any changes that were made while in mode 4.